go/host/l1/dataservice.go

```diff
@@ -303,6 +303,8 @@ func (r *DataService) processRollupLogs(l types.Log, txData *common.L1TxData, pr
 			},
 		}
 		processed.AddEvent(common.RollupTx, txData)
+	} else {
+		r.logger.Debug(fmt.Sprintf("error while fetching blobs. Cause: %s", err))
```

Comment:
Actually this is no good, the err is defined in the if statement so it's out of scope at this point, you'll have the nil value from earlier on. Need to take that FetchBlobs call out of the if statement and do it in the natural way I think